.github/workflows/nightly_vtk_master.yml

```diff
@@ -132,15 +133,44 @@ jobs:
         lfs_sha: ${{ needs.cache_lfs.outputs.lfs_sha}}
 
 #----------------------------------------------------------------------------
-# MacOS CI: Build and test, cross-cpu build matrix
+# MacOS CI: Build and test
 #----------------------------------------------------------------------------
   macos:
     needs: [cache_lfs, check_nightly]
     if: ${{ needs.check_nightly.outputs.should_run == 'true' }}
     strategy:
       fail-fast: false
       matrix:
-        cpu: [x86_64, arm64]
+        cpu: [x86_64]
```

Comment:
should we remove the matrix field and just set the cpu directly in `with` on both jobs?